### DISCUSSION TITLE
Where folke's YT channel ?

### DISCUSSION THREAD

**Discussion by radvil** [LazyVim/LazyVim#3463] [General]
I found this on facebook, Where folke's YT channel ? or is this from TJ ?
![image](https://github.com/LazyVim/LazyVim/assets/36059968/67796acf-b974-47cf-9f38-473f4b11bcf5)

**Comment by dpetka2001** [comment]
If you manage to get the link of this video, pls do share it here.

**Reply by dpetka2001** [reply]
Probably this [one](https://www.youtube.com/watch?v=3ir7fHwwoJs)?

**Reply by radvil** [reply]
I think that's the one on the screenshot, so it was from TJ's stream ... When will folke go on stream again ??

**Reply by dpetka2001** [reply]
I'm not the right person to ask this???

**Reply by radvil** [reply]
well, I hope TJ will ask him for neovim stream again.

**Comment by radvil** [comment]
I found channel, but I am afraid it is a fake folke, because no vim videos, only DJs wtf
![image](https://github.com/LazyVim/LazyVim/assets/36059968/6a873d74-3eee-46b9-9ee9-5e46b24a7fca)

**Reply by folke** [reply]
That's me, I just don't do YT

**Reply by folke** [reply]
I also had no idea I had a public profile with some of my playlists on it. 
Just made them private...

**Reply by radvil** [reply]
Any chance to go on TJ's livestream again? Please let us know if so😁

**Comment by nipuna-perera** [comment]
I know you don't want to do YT and I respect that. But if you ever decide
to stream, even if it's just working on your plugins with no voice I'd
definitely watch it. I'm sure there are a ton of others that will too.

**Reply by radvil** [reply]
I second this🤟
